deployments/infrastructure-target-aws.nix

```diff
@@ -33,11 +39,54 @@ rec {
     };
 
     deployment.ec2 = {
-      accessKeyId = IOHKaccessKeyId;
-
+      inherit accessKeyId;
+      elasticIPv4 = resources.elasticIPs.cardanod-ip;
       instanceType = mkForce "r3.2xlarge";
       ebsInitialRootDiskSize = mkForce 50;
       associatePublicIpAddress = true;
+      securityGroups = [
+        resources.ec2SecurityGroups."allow-all-ssh-${region}-${org}"
+      ];
+    };
+  };
+
+  resources = {
+    elasticIPs = {
+      hydra-ip    = { inherit region accessKeyId; };
+      cardanod-ip = { inherit region accessKeyId; };
+    };
+    ec2SecurityGroups = {
+      "allow-deployer-ssh-${region}-${org}" = {
+        inherit region accessKeyId;
+        description = "SSH";
+        rules = [{
+          protocol = "tcp"; # TCP
+          fromPort = 22; toPort = 22;
+          sourceIp = deployerIP + "/32";
+        }];
+      };
+      "allow-all-ssh-${region}-${org}" = {
+        inherit region accessKeyId;
+        description = "SSH";
+        rules = [{
+          protocol = "tcp"; # TCP
+          fromPort = 22; toPort = 22;
+          sourceIp = "0.0.0.0/0";
+        }];
+      };
+      "allow-public-www-${region}-${org}" = {
+        inherit region accessKeyId;
+        description = "WWW";
+        rules = [{
+          protocol = "tcp"; # TCP
+          fromPort = 443; toPort = 443;
+          sourceIp = "0.0.0.0/0";
+        }];
+      };
     };
+    datadogMonitors = (with (import ./../modules/datadog-monitors.nix); {
```

Comment:
This should be in production config only

Reply:
Yeah, that slipped..

Reply:
Fixed that, and also moved tarsnap defns to the common `infrastructure.nix`.